docs/solutions/reference-designs/eval-cn0584-ebz/index.rst

```diff
@@ -0,0 +1,852 @@
+accurate, 20ppm/°C max voltage reference. For more stringent use cases where
+the accuracy and temperature drift is an issue, an external :adi:`LTC6655`
+2.048V, ±0.025% accurate, 2ppm/°C max voltage reference can be used.
+
+The default DAC reference configuration uses the internal 2.5V, ±0.3% accurate,
+10ppm/°C max voltage reference. For more stringent use cases where the accuracy
+and temperature drift is an issue, an external :adi:`ADR4525` 2.5V,
+±0.02% accurate, 2ppm/°C max voltage reference can be used.
+
+.. table:: VREF Configuration
+
+   +------------------+-------------------------+
+   | VREF             | Jumper Settings         |
+   +==================+=========================+
+   | ADC_VREF         | Short P5                |
+   +------------------+-------------------------+
+   | DAC_VREF         | Short P4                |
+   +------------------+-------------------------+
+
+Power Supply Considerations and Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All power for CN0584 is provided by :adi:`EVAL-CN0585-FMCZ <CN0585>` through
+the analog front end (AFE) connector. CN0584 uses the +15V and -15V rails to provide the positive
+and negative supply voltages for the :adi:`ADG5421F` input protection switches. The +12V and -12V rails 
+provide the positive and negative supply voltages for the :adi:`ADA4898-1` ADC buffer amplifiers. 
+The +3.3V rail powers the EEPROM circuit. 
+
+Power tree information can be found in :adi:`EVAL-CN0585-FMCZ <CN0585>`. 
+Table 6 provides more details on LLDK power rails:
+
+.. table:: Power Rail Descriptions
+
+   ========== ============================================================
+   Power Rail Description
+   ========== ============================================================
+   +12 V      LT3045-1 provides the 12V rail supplying up 280mA
+   -12 V      LT3094 provides the -12V rail supplying up to -280mA
+   +15 V      LTM8049 provides the +15V rail at 80% efficiency
+   -15 V      LTM8049 provides the -15V rail at 80% efficiency
+   +3.3 V     Fed through from the FPGA FMC connector to the AFE connector
+   ========== ============================================================
+
+System Setup Using a ZedBoard
+-----------------------------
+
+CN0584 is fully supported using a ZedBoard.
+
+.. figure:: eval-cn0585-fmcz_kit_top.jpg
+   
+   EVAL-CN0585-FMCZ revB connected to EVAL-CN0584-EBZ
+
+.. figure:: eval-cn0585-fmcz_angle-web.jpg
+   
+   EVAL-CN0585-FMCZ revB
+
+The following is a list of items needed for system setup:
+
+- **Hardware**
+
+  - :adi:`EVAL-CN0585-FMCZ(Rev B) <CN0585>` 
+       (Note: Figure 4 features EVAL-CN0585-FMCZ RevA board. Current LLDK system has RevB board shown in
+       Figure 5, same connectors and functionalities, only different in USB-C power supply.)
+  - :adi:`EVAL-CN0584-EBZ <CN0584>`
+  - `ZedBoard <https://www.avnet.com/wps/portal/us/products/avnet-boards/avnet-board-families/zedboard/>`__
```

Comment:
@jvelasc2  can you change the URL to https://digilent.com/reference/programmable-logic/zedboard/start